app/config/FrontendAppConfig.scala

```diff
@@ -53,7 +53,8 @@ class FrontendAppConfig @Inject()(val servicesConfig: ServicesConfig, val config
   //Individual Calculation Service
   lazy val individualCalculationsService: String = servicesConfig.baseUrl("individual-calculations")
 
-
+	//View L&P
+	def saViewLandPService(utr: String): String = servicesConfig.getString("old-sa-viewer-frontend.host") + s"/$utr/account"
```

Comment:
URL in Jira is different i.e., https://www.tax.service.gov.uk/self-assessment/ind/insert/UTR/account